Library/Formula/ledger.rb

```diff
@@ -20,7 +18,8 @@ class Ledger < Formula
 
   head "https://github.com/ledger/ledger.git"
 
-  option "debug", "Build with debugging symbols enabled"
+  option "with-debug", "Build with debugging symbols enabled"
+  option "with-check", "Run build-time check"
```

Comment:
Currently `acprep check` will fail sometimes, I'm working on fixing the issue upstream and I'd prefer to change it once the tests pass reliably.